python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -624,8 +755,9 @@ def assembleMetadata(self, coaddExposure, tempExpRefList, weightList):
         # (and we need more than just the PropertySet that contains the header), which is not possible
         # with the current butler (see #2777).
         tempExpList = [tempExpRef.get(tempExpName + "_sub",
-                                      bbox=afwGeom.Box2I(afwGeom.Point2I(0, 0), afwGeom.Extent2I(1, 1)),
-                                      imageOrigin="LOCAL", immediate=True) for tempExpRef in tempExpRefList]
+                                      bbox=afwGeom.Box2I(coaddExposure.getBBox().getMin(),
+                                                         afwGeom.Extent2I(1, 1)), immediate=True)
```

Comment:
`ShimButler`'s  `_sub` bypass implementation currently doesn't take `imageOrigin="LOCAL"`.   If you think you'll need that, we could file a ticket and @TallJimbo can add it. For this use case, the workaround didn't seem any more complex.

Reply:
[DM-17416](https://jira.lsstcorp.org/browse/DM-17416).  I had actually tried to support this, but got the parameter name wrong ("origin" instead of "imageOrigin").  But no objection to the workaround.

As a side note, once we don't need to worry about backwards compatibility, the single-pixel trick should be unnecessary, as in Gen3 you should be able to `get` all Exposure components directly.